### PR TITLE
Added -K option to pg_verifybackup command

### PR DESCRIPTION
**Pull request by s-hironobu**
I think  pg_verifybackup command needs -K option, so I added it.

### CODE REVIEW

File: src/bin/pg_verifybackup/pg_verifybackup.c [hunk 1]
```diff
@@ -793,12 +816,31 @@ parse_required_wal(verifier_context *context, char *pg_waldump_path,
...
+									  (uint32) (this_wal_range->end_lsn >> 32),
+									  (uint32) this_wal_range->end_lsn);
+#else
 		pg_waldump_cmd = psprintf("\"%s\" --quiet --path=\"%s\" --timeline=%u --start=%X/%X --end=%X/%X\n",
 								  pg_waldump_path, wal_directory, this_wal_range->tli,
 								  (uint32) (this_wal_range->start_lsn >> 32),
 								  (uint32) this_wal_range->start_lsn,
 								  (uint32) (this_wal_range->end_lsn >> 32),
 								  (uint32) this_wal_range->end_lsn);
+#endif
```
Comment (laurenz) [inline src/bin/pg_verifybackup/pg_verifybackup.c:843]: Couldn't that be simplified to:

```c
#ifdef	USE_ENCRYPTION
		if (data_encrypted)
			pg_waldump_cmd = psprintf("\"%s\" --quiet --path=\"%s\" --timeline=%u --start=%X/%X --end=%X/%X" \
									  " -K %s\n",
									  pg_waldump_path, wal_directory, this_wal_range->tli,
									  (uint32) (this_wal_range->start_lsn >> 32),
									  (uint32) this_wal_range->start_lsn,
									  (uint32) (this_wal_range->end_lsn >> 32),
									  (uint32) this_wal_range->end_lsn,
									  encryption_key_command);
		else
#endif
		pg_waldump_cmd = psprintf("\"%s\" --quiet --path=\"%s\" --timeline=%u --start=%X/%X --end=%X/%X\n",
								  pg_waldump_path, wal_directory, this_wal_range->tli,
								  (uint32) (this_wal_range->start_lsn >> 32),
								  (uint32) this_wal_range->start_lsn,
								  (uint32) (this_wal_range->end_lsn >> 32),
								  (uint32) this_wal_range->end_lsn);
```

File: src/bin/pg_verifybackup/pg_verifybackup.c [hunk 2]
```diff
@@ -3,7 +3,7 @@
  * pg_verifybackup.c
  *	  Verify a backup against a backup manifest.
  *
- * Portions Copyright (c) 1996-2020, PostgreSQL Global Development Group
+ * Portions Copyright (c) 1996-2021, PostgreSQL Global Development Group
```
Comment (laurenz) [inline src/bin/pg_verifybackup/pg_verifybackup.c:6]: Are we in the business of updating the PGDG copyright?
We took the source from 2020.

**Review by laurenz — COMMENTED** [review]
Other that these two nits, the change looks alright.

**Comment by s-hironobu** [comment]
I close this pull-request because I will send a new pull-request that includes this.

**Comment by laurenz** [comment]
You wouldn't have had to close the pull request - all future commits on the same branch automatically get added to the existing one.

**Pull request closed without merging**